### PR TITLE
Mail sending error

### LINKED ISSUE #18: Exception on create, unknown attribute, first_name
```
2016-02-19T20:56:36.229662+00:00 app[web.1]: Started POST "/users" for 216.127.112.163 at 2016-02-19 20:56:36 +0000
2016-02-19T20:56:36.274506+00:00 app[web.1]:   Parameters: {"utf8"=>"✓", "authenticity_token"=>"VA9JTDfk1fceKSLmB3pSMsTweCTtmRkYANevrvCQgkc=", "user"=>{"first_name"=>"John", "last_name"=>"Chapin", "email"=>"john@johnchapin.net", "receipt"=>#<ActionDispatch::Http::UploadedFile:0x007f5b4c162e30 @original_filename="ivocate-logo.png", @content_type="image/png", @headers="Content-Disposition: form-data; name=\"user[receipt]\"; filename=\"ivocate-logo.png\"\r\nContent-Type: image/png\r\n", @tempfile=#<Tempfile:/tmp/RackMultipart20160219-12-3zse2o>>}, "commit"=>"Sign Up Now"}
2016-02-19T20:56:36.376740+00:00 app[web.1]: 
2016-02-19T20:56:36.376744+00:00 app[web.1]:   app/controllers/users_controller.rb:34:in `create'
2016-02-19T20:56:36.376743+00:00 app[web.1]: ActiveRecord::UnknownAttributeError (unknown attribute: first_name):
2016-02-19T20:56:36.376745+00:00 app[web.1]: 
2016-02-19T20:56:36.376745+00:00 app[web.1]: 
2016-02-19T20:56:36.273059+00:00 app[web.1]: Processing by UsersController#create as HTML
2016-02-19T20:56:36.374394+00:00 app[web.1]: Completed 500 Internal Server Error in 100ms
```

### PR DESCRIPTION
**Pull request by jchapin**
Fixes #18

### CODE REVIEW

File: app/views/user_mailer/signup_email.html.erb [hunk 1]
```diff
@@ -1,82 +1,13 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (jchapin) [inline app/views/user_mailer/signup_email.html.erb]: The e-mail was throwing errors, and considering that most of this content was about shaving and was going to get pitched, I stripped it out.

File: app/models/user.rb [hunk 2]
```diff
@@ -1,58 +1,60 @@
...
     end
 
-    def send_welcome_email
-        UserMailer.delay.signup_email(self)
-    end
+    self.referral_code = referral_code
+  end
+
+  def send_welcome_email
+    UserMailer.signup_email(self).deliver
```
Comment (jchapin) [inline app/models/user.rb]: Switched from delay to deliver on the spot.

File: app/views/users/refer.html.erb [hunk 3]
```diff
@@ -74,38 +74,13 @@
...
-              <% end %>
-          </ul>
-
-          <div class="progress">
-              <div class="mover">
-                  <div class="bar clearfix"></div>
-              </div>
-          </div>
--->
-
```
Comment (jchapin) [inline app/views/users/refer.html.erb]: Strip out commented out code.

File: config/environments/production.rb [hunk 4]
```diff
@@ -73,6 +75,21 @@
...
+    authentication: :plain,
+    user_name: ENV['SENDGRID_USERNAME'],
+    password: ENV['SENDGRID_PASSWORD'],
+    domain: 'heroku.com',
+    enable_starttls_auto: true
+  }
+  config.action_mailer.default_url_options = {
+    host: 'www.themindfulnessedge.com'
+  }
+
```
Comment (jchapin) [inline config/environments/production.rb]: Added SendGrid config.

File: Gemfile [hunk 5]
```diff
@@ -10,7 +11,7 @@ gem 'json', '~> 1.8.2'
 gem 'kgio', '~> 2.10.0'
 gem 'activeadmin'
 gem "paperclip", "~> 4.3"
-gem 'aws-sdk'
+gem 'aws-sdk', '< 2.0'
```
Comment (jchapin) [inline Gemfile]: Apparently we're going to be stuck on pre-2.0 AWS SDK for little while longer:
http://stackoverflow.com/questions/28374401/nameerror-uninitialized-constant-paperclipstorages3aws

**Comment by jchapin** [comment]
@RanSolo, @cbradshaw ready for review. This now has the production servers sending out e-mail: http://ni-prelaunchr-dev.herokuapp.com

File: app/models/user.rb (hunk 2 above)
Comment (RanSolo) [inline app/models/user.rb]: :+1:

**Comment by RanSolo** [comment]
:+1: 
Added sendgrid
Switched to deliver on the spot.
Stripped out all shaving stuff and commented out code

**Pull request merged**